content/pages/airport-cases.md

```diff
@@ -0,0 +1,41 @@
+Title: The Airport Cases
+Date: 2017-01-31 **:**
+Author: anseljh
+Slug: airport-cases
+Status: draft
+
+On January 27, 2017, President Trump signed an [Executive Order titled "Protecting the Nation from Foreign Terrorist
+Entry Into the United States"](https://assets.documentcloud.org/documents/3431047/Extreme-Vetting-EO.pdf). The next day, civil rights and immigration lawyers began filing legal challenges throughout the country. As the dockets from these cases become available in Free Law Project's [RECAP Archive](https://www.courtlistener.com/recap/), we are linking them on this page.
```

Comment:
@mlissner Where should the PDF go? In `/content/pages/featured-cases/`, or in `/content/pdf/`, or elsewhere?